scripts/download_wasm.py

```diff
@@ -56,7 +56,7 @@ def download_wasm(name: str, version: str, destination: str) -> bool:
 
 
 if __name__ == '__main__':
-    ret = download_wasm('findex', 'v2.0.1', 'src/pkg')
+    ret = download_wasm('findex', 'last_build', 'src/pkg')
```

Comment:
my bad I should have added a comment: we shouldn't touch this version (or only set the next release version number (`v2.0.2`?))

Reply:
I touch it only to have the CI pass (or fail and see the errors) but it doesn't work since we have two PR under work on Findex anyway